### ISSUE TITLE
server browser

### ISSUE THREAD

**Issue by mint-xoxo** [SwagSoftware/Kisak-Strike#13]
possibly fix server browser

**Comment by LWSS** [comment]
There's no point in doing this until you can actually connect to servers. 

I think I might need a reservation cookie from the GC still, I don't remember.

Version spoof can be done with steam.inf

**Comment by mint-xoxo** [comment]
yeah i was thinking bout the version issues when i was writing that
